src/core/js/models/itemsQuestionModel.js

```diff
@@ -215,4 +216,51 @@ export default class ItemsQuestionModel extends BlendedItemsComponentQuestionMod
     return 'choice';
   }
 
+  /**
+   * Creates a string explaining the answer(s) the learner should have chosen
+   * Used by ButtonsView to retrieve question-specific correct answer text for the ARIA
+   * 'live region' that gets updated when the learner selects the 'show correct answer' button
+   * @return {string}
+   */
+  getCorrectAnswerAsText() {
+    const globals = Adapt.course.get('_globals')._components['_' + this.get('_component')];
+    let ariaAnswer;
+    let correctAnswer;
+
+    if (this.isSingleSelect()) {
+      ariaAnswer = globals.ariaCorrectAnswer;
+      const correctOption = this.getChildren().findWhere({ _shouldBeSelected: true });
+      correctAnswer = correctOption.get('text');
+    } else {
+      ariaAnswer = globals.ariaCorrectAnswers;
+      const correctOptions = this.getChildren().where({ _shouldBeSelected: true });
+      correctAnswer = correctOptions.map(correctOption => correctOption.get('text')).join('<br>');
```

Comment:
As discussed, could always pass through the correct answers as an array, giving flexibility to the plugin globals:
```hbs
The correct answers are {​{​#each correctAnswer}​}​{​{​{​this}​}​}​{​{​#unless @last}​}​<br>{{/unless}}{​{​/each}​}​
```
Don't know if it will ever be tweaked in practice.

Reply:
It's a nice suggestion but as you say, whether it will ever get used in practice is questionable.... can easily be changed in future if it does come up as a requirement.